### PR TITLE
Bug fix - Error of calculation of rates in diagnostics

### PR DESCRIPTION
**Pull request by patapufate**
This PR fixes the calculation of the minimum and maximum rate in the diagnostic parameters.

### CODE REVIEW

File: vectornav/src/main.cpp [hunk 1]
```diff
@@ -334,9 +334,9 @@ int run(int argc, char *argv[])
     int async_output_rate;
 
     // Sensor IMURATE (800Hz by default, used to configure device)
-    int SensorImuRate;
-    int minImuRate;
-    int maxImuRate;
+    double SensorImuRate;
+    double minImuRate;
+    double maxImuRate;
```
Comment (hidmic) [inline vectornav/src/main.cpp:339]: @patapufate I think we should fix this. IIRC the IMU rate is the sampling rate, not the output or publishing rate. The latter is called `async_output_rate`. We should change our diagnostic check parameters to match that e.g.: `max_async_output_rate` and `min_async_output_rate`. It'll be less confusing.

**Review by hidmic — COMMENTED** [review]
Oof, thanks for the fix @patapufate.

File: vectornav/params/vn100.yaml [hunk 2]
```diff
@@ -8,12 +8,12 @@ serial_baud: 115200
 # Acceptable data rates in Hz: 1, 2, 4, 5, 10, 20, 25, 40, 50, 100, 200
 # Baud rate must be able to handle the data rate
 async_output_rate: 25
+min_async_output_rate: 12.5
+max_async_output_rate: 50
```
Comment (hidmic) [inline vectornav/params/vn100.yaml:12]: @patapufate +1, but will it still be possible to tweak these from `system_testbed` tests? Tests need the ability to set (bad) rates.

**Review by hidmic — COMMENTED** [review]
@patapufate consider not squashing on merge, so that we get to keep both commits.

File: vectornav/params/vn100.yaml (hunk 2 above)
Comment (patapufate) [reply vectornav/params/vn100.yaml:12]: Checked.

**Review by hidmic — APPROVED** [review]
LGTM !

**Pull request merged**